src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -1843,7 +1843,7 @@ class BGPConfigDaemon:
     cmn_key_map = [('asn&peer_type',                        '{no:no-prefix}neighbor {} remote-as {}'),
                    (['local_asn', '+local_as_no_prepend',
                      '+local_as_replace_as'],               '{no:no-prefix}neighbor {} local-as {} {:no-prepend} {:replace-as}'),
-                   (['admin_status', '+shutdown_message'],  '{no:no-prefix}neighbor {} shutdown {:shutdown-msg}', ['false', 'true']),
```

Comment:
I understand you are making this change for consistency; did you try loading old configs with this change? see if this breaks backward compatibility?

Reply:
Yes, I did verify that when a config is loaded at boot or via config reload, a value of true/false still works as it previously did.  However, it does not work when the config is updated at runtime, which is to be expected.  Infact when I tested that I got yang validation failures that prevented it due to yang expecting up/down.

Reply:
OK, thanks!